src/test/java/operationsTest.java

```diff
@@ -0,0 +1,11 @@
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class operationsTest {
+    @Test
+    //dummy testcase
+    void contstactorTest() {
```

Comment:
Ask Yahya and goda about mockito function for test. This is not a pr blocker for this pr

Reply:
ok